src/healthkit.cs

```diff
@@ -189,6 +189,18 @@ interface HKPredicateKeyPath {
 		[Watch (4, 0), iOS (11, 0)]
 		[Field ("HKPredicateKeyPathWorkoutTotalFlightsClimbed")]
 		NSString TotalFlightsClimbed { get; }
+
+		[Watch (5, 0), iOS (12, 0)]
+		[Field ("HKPredicateKeyPathSum")]
+		NSString PathSum { get; }
+
+		[NoWatch, iOS (12, 0)]
+		[Field ("HKPredicateKeyPathClinicalRecordFHIRResourceIdentifier")]
+		NSString ClinicalRecordFHIRResourceIdentifier { get; }
```

Comment:
`ClinicalRecordFHIRResourceIdentifier` -> `ClinicalRecordFhirResourceIdentifier`

Reply:
Same as with the other property. Lets look at other API and be consistent.